### ISSUE TITLE
Please pick the version you prefer for Coq 8.18 in Coq Platform 2023.10

### ISSUE THREAD

**Issue by MSoegtropIMC** [math-comp/math-comp#1089]
The Coq team released Coq `8.18.0` on September 7th, 2023.
The corresponding Coq Platform release `2023.10` should be released before **November 30th, 2023**.
It can be delayed in case of difficulties until January 15th, 2023, but this should be an exception.

This issue is to inform you that the opam package we are currently testing in Coq Platform CI **works fine** with Coq `8.18.0`.

Coq Platform CI is currently testing opam package `coq-mathcomp-ssreflect.2.0.0`
from official repository https://coq.inria.fr/opam/released/packages/coq-mathcomp-ssreflect/coq-mathcomp-ssreflect.2.0.0/opam. 

**In case this is the version you want to see in Coq Platform, there is nothing to do for you - please just close this issue.**

Please note that this pick also applies to:
```
coq-mathcomp-ssreflect.2.0.0
coq-mathcomp-fingroup.2.0.0
coq-mathcomp-algebra.2.0.0
coq-mathcomp-solvable.2.0.0
coq-mathcomp-field.2.0.0
coq-mathcomp-character.2.0.0
coq-mathcomp-finmap.2.0.0
coq-mathcomp-real-closed.2.0.0
coq-mathcomp-multinomials.2.0.0
```

In case you would prefer to see an updated or an older version in the upcoming Coq Platform `2023.10`, please inform us as soon as possible and before **October 31st, 2023**!

The working branch of Coq Platform, can be found here [main](https://github.com/coq/platform/tree/main).
It contains package pick [`~8.18+beta1`](https://github.com/coq/platform/tree/main/package_picks/package-pick-8.18+beta1.sh) which already supports Coq version `8.18.0` and contains already working (possibly patched / commit pinned) Coq Platform packages.

In case you want to select a different version, please **don't** close this issue, even after creating the new tag and/or opam package.
We will close the issue after updating Coq Platform.
This is a measure of 'double book keeping' in order to avoid that we ship the wrong version.

In any case, Coq Platform won't be released before this issue is closed!

Thanks!

P.S.: this issue has been created automatically based on CI status.

CC: https://github.com/coq/platform/issues/372

**Comment by proux01** [comment]
@MSoegtropIMC 2.0 would be nice but there are still currently a few unported things (mostly Analysis, Abel, Jasmin) still relying on 1.17, so don't know what's best (I'd say I'm trusting you here, so if you think 2.0 is fine, let's go for it).

Also note that we plan to release 1.18 and 2.1 by the end of the month.

**Comment by MSoegtropIMC** [comment]
We still have some time for the release. I am doing the "please pick/tag" issues today and will collect maintainer feedback for all packages which don't work with mathcomp 2.0. In case I don't get feedback in due time, I might ask you for help following up with the maintainers. As soon as we have official feedback, we can discuss what we do.

The tracker issue is linked above.

**Comment by proux01** [comment]
@MSoegtropIMC if possible yes using 2.1.0 would be great (but if this requires too much OPAM packages adaptation, 2.0.0 is fine)

(About mathcomp 1, we plan to release 1.18 next week.)

**Comment by proux01** [comment]
@MSoegtropIMC most OPAM packages working with mathcomp 2.0.0 should now work for 2.1.0 (most of the time this was achieved by just bumping the bound in the OPAM file, only multinomials and fourcolor required a new release (resp. 2.1.0 and 1.3.1))

**Comment by MSoegtropIMC** [comment]
As discussed in [Zulip](https://coq.zulipchat.com/#narrow/stream/237666-math-comp-analysis/topic/release.200.2E6.2E6) mathcomp-analysis won't support mathcomp 2.0 before the end of the year, so the conclusion is that the Coq Platform release will include two picks, one based on matchcomp 1.18 and one based on 2.0 (without mathcomp-analysis).

There will be two installers for Mac and Windows. snap does not support this easily, so I would suggest that on snap we use 1.18 for the release and 2.0 for a beta in the 8.18 track.

Is this agreeable? If not, please suggest another solution.

**Comment by proux01** [comment]
LGTM, thanks for your efforts!

Note that 2.1 should support all platform packages IINM, but 2.0 is fine too.

**Comment by MSoegtropIMC** [comment]
Indeed I missed the 2.1 release. If you prefer 2.1, I can try to update.

**Comment by MSoegtropIMC** [comment]
And I guess with "all platform packages" you mean all except coq-mathcomp-analysis.

**Comment by MSoegtropIMC** [comment]
@proux01: I tried this package selection (instead of all 2.0.0) and it also works:
```
coq-mathcomp-ssreflect.2.1.0
coq-mathcomp-fingroup.2.1.0
coq-mathcomp-algebra.2.1.0
coq-mathcomp-solvable.2.1.0
coq-mathcomp-field.2.1.0
coq-mathcomp-character.2.1.0
coq-mathcomp-bigenough.1.0.1
coq-mathcomp-finmap.2.0.0
coq-mathcomp-real-closed.2.0.0
coq-mathcomp-zify.1.5.0+2.0+8.16
coq-mathcomp-multinomials.2.1.0
```
Note that for finmap and real-closed there is no 2.1 as yet.

It is your choice - I can use either version.

**Comment by proux01** [comment]
@MSoegtropIMC thanks, sounds good

> Note that for finmap and real-closed there is no 2.1 as yet.

Yes, no need for it since their 2.0 version works with mathcomp 2.1

**Comment by MSoegtropIMC** [comment]
OK, I go with the above list for Coq Platform 2023.10 then. I will close this issue a bit later today.
